Realm/RLMApp.mm

```diff
@@ -334,16 +330,16 @@ - (instancetype)initWithApp:(std::shared_ptr<realm::app::App>&&)app config:(RLMA
 - (instancetype)initWithConfiguration:(RLMAppConfiguration *)configuration {
     if (self = [super init]) {
         _app = RLMTranslateError([&] {
-            return app::App::get_app(app::App::CacheMode::Enabled, configuration.config, configuration.clientConfig);
+            return app::App::get_app(app::App::CacheMode::Disabled, configuration.config);
```

Comment:
Why are we disabling the caching here? Is it because we're anyway maintaining an SDK-level cache?

Reply:
Yeah, we never actually get a cache hit in the core cache unless something has gone wrong.